docs/en/observability/synthetics-quickstart-fleet.asciidoc

```diff
@@ -50,16 +50,14 @@ You can use the _Enabled_ toggle to turn the monitor on or off in {fleet} withou
 [role="screenshot"]
 image::images/synthetics-integration.png[Synthetics integration]
 
-Set the _Source Type_ for the browser monitor.
-If you're using a <<synthetics-test-suite, synthetic test suite>>, use the _Zip URL_ type.
+Set the *Source Type* for the browser monitor.
```

Comment:
There are a couple of other places where italics are used instead of bold (can't mark them because they aren't in the diff). Might be a good idea to clean them up while you have the file open. Otherwise, looks great!

Reply:
nevermind...I think you've covered those in another PR. :-)